projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/notification/EmailNotification.java

```diff
@@ -5,18 +5,7 @@
 
```

Comment:
This is actually a new interface. But the interface just so happens to have the name of hte previous class.

Reply:
what do you mean add as a comment to the code. 
After this is merged why would ppl want to know that?